modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
I mean we should have physical access for the next few months, and the VMs can be accessed with VNC too, so I don't see this as something that needs to be running straight off. It would be nice though to have running now, so maybe add `ensure => running` for now and put a TODO to remove it later? I don't think I have a better solution than that.